test/test_helper.rb

```diff
@@ -331,6 +331,15 @@ def wait_for_ajax
         loop until page.evaluate_script('jQuery.active').zero?
       end
     end
+
+    def wait_for_orchestration_requests
+      Timeout.timeout(Capybara.default_max_wait_time * 2) do
+        # Notice host-progress won't be visible without a multi-threaded server
+        loop { find("#host-progress", :visible => false, :wait => 0.1) }
```

Comment:
Is this whole loop equivalent to `assert has_no_selector?`?  It just waits until the `#host-progress` element is no longer present.